docs/intro/tutorial08.txt

```diff
@@ -48,12 +48,15 @@ The steps are not duplicated in this tutorial, because as a third-party
 package, it may change separately to Django's schedule.
 
 Once installed, you should be able to see the DjDT "handle" on the right side
-of the browser window when you refresh the polls application. Click it to open
-the debug toolbar and use the tools in each panel. See the `panels
-documentation page
-<https://django-debug-toolbar.readthedocs.io/en/latest/panels.html>`__ for more
+of the browser window when you browse to the admin of your project (http://localhost:8000/admin). 
+Click it to open the debug toolbar and use the tools in each panel. 
+See the `panels documentation page <https://django-debug-toolbar.readthedocs.io/en/latest/panels.html>`__ for more
 information on what the panels show.
 
+To see the toolbar in the polls application make sure all your templates
+are using `complete HTML documents <https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started#anatomy_of_an_html_document>`_.
+The toolbar will only render in responses that contain the ``<body></body>`` tags.
```

Comment:
We can have this in an `... admonition::` or we can remove this paragraph entirely?
In your reasoning docs, you mention
> The main goal of this section is to learn how to add a package in the first place and a bit of django-debug-toolbar in second place.

It's a good point, after telling them they can see this on the polls app we don't encourage them to do anything with this information (such as checking a query).
This information is certainly secondary (they don't have to do this) and possibly not worth the headspace? What do you think?

Reply:
Done in https://github.com/django/django/pull/18285/commits/d5133759f662f0c6b6a90ed3c904fa16c4c5ed70

Reply:
> I think it's a reasonable question for people to ask when going through the tutorial of "Why does the toolbar not show up for the application I've built?". The toolbar doesn't make it super obvious that you need a </body> tag since you need to go to the reference documentation to find that.

I agree it's a reasonable question. 
I like to think of it this way... for this tutorial, which is often the first time someone uses Django, we are designing the journey for the reader. In this designed journey, they don't need to worry. The less information/worries the better

Curious readers can google and explore on their own

Reply:
I understand the reasoning to keep the tutorials/documentation as slim as possible and people can eventually figure out why. I don't agree with it though because it's not a clear reason that can be quickly searched for on the internet.

Reply:
Should be an admonition for sure 

My first google of "why isn't django-debug-toolbar showing" had the relevant docs from django-debug-toolbar appear 🤷‍♀️ but I know the issue and what the fix is. Maybe other people would google differently 

Saw someone on reddit ask this question of the tutorial and there's the ticket raiser too, but we previously told them to go to the polls application and that it would be visible there. Hard to know if anyone would do that without us telling them to

Reply:
If you'd like to leave it off here, I'll add it to the toolbar's docs to explain what's going on.

Reply:
Ok let's proceed with this now, an admonition can be added at a later date if required 👍